### ISSUE TITLE
Activation failed: Not all requested modules can be enabled

### ISSUE THREAD

**Issue by BillyGeat** [raydac/netbeans-mmd-plugin#5]
Hi!

After upgrading to Netbeans 8.1, I imported all my installed add-ons and the mind map add-on does not start. I get this message:

> Activation failed: Not all requested modules can be enabled: > [StandardModule:com.igormaznitsa.nbmindmap jarFile: C:\Users\billy\AppData\Roaming\NetBeans\8.1\modules\com-igormaznitsa-nbmindmap.jar]

I have installed the PHP-Version of Netbeans. I don't know if it needs the full version?! Should not I think...

**Comment by raydac** [comment]
try to reinstall (uninstall and then install) the mind map plugin, looks like that the issue for migration process

**Comment by BillyGeat** [comment]
That is the solution, thank you!

I have noticed during the installation process, that I had to accept much more "legal stuff" than the license for the mind map add-on (all of them was java related).
It looks like this add-on depends on many java stuff, that is not part of a standard php-netbeans-version installation. But the add-on installer does all needed steps to make it work.

Only a migration does not work.
